packages/dev/core/src/Cameras/freeCamera.ts

```diff
@@ -376,36 +376,47 @@ export class FreeCamera extends TargetCamera {
             actualDisplacement = displacement.add(this.getScene().gravity);
         }
 
-        coordinator.getNewPosition(this._oldPosition, actualDisplacement, this._collider, 3, null, this._onCollisionPositionChange, this.uniqueId);
+        coordinator.getNewPosition(
+            this._oldPosition,
+            actualDisplacement,
+            this._collider,
+            3,
+            null,
+            (_collisionId: number, newPosition: Vector3, collidedMesh: Nullable<AbstractMesh> = null) => {
+                const updatePosition = (newPos: Vector3) => {

```

Comment:
Here as well, not sure to understand why we need to define updatePosition and use it here only once ?

Reply:
This code existed already, I didn't change it. I agree it is not needed :-) 
I can fix that